deployer/aws-lambda/content-origin-response/index.js

```diff
@@ -14,6 +14,16 @@ exports.handler = async (event) => {
   const response = event.Records[0].cf.response;
   const uri = request.uri.toLowerCase();
 
+  // Prior to May 2021, we used to host the live samples like this:
+  //   /en-US/docs/Web/Foo/_samples_/SampleID/index.html
+  // But then, in https://github.com/mdn/yari/pull/3798, we change it to:
+  //   /en-US/docs/Web/Foo/_sample_.SampleID.html
+  // So to make deployment-timing easier we make this code here work for
+  // both the old way and the new way.
+  // (Later in 2021 we can remove any mentions of `/_samples_/`)
+  const isLiveSamplesURI =
+    uri.includes("/_samples_/") || uri.includes("/_sample_.");
+
```

Comment:
@Gregoor @escattone Turns out, I did want to kill two birds with one stone here. 
Not ideal, but I thought about the fact that we had had a Zoom conversation which is beyond the norm. 

Now, the way the `content-origin-response` Lambda@Edge function can figure out if we're viewing a live sample and if that's the case, ...

1. Don't set the `X-Frame-Options` header
2. Don't set the CSP header

r?